iwp_website/templates/loan_issue.xml

```diff
@@ -16,9 +16,9 @@
                 required="required">
                 <option t-foreach="loan_issues or []" t-as="li"
                     t-att-selected="'selected' if li.id == loan_issue else None"
-                    t-att-data-facial_value="li.face_value"
+                    t-att-data-face_value="li.face_value"
```

Comment:
I think this was changed back to facial values.

Reply:
Pour l'instant ça résoud juste un bug. J'avais nommé `face_value` à un endroit et `facial_value` à un autre.

Si ça a changé dans une autre PR, j'irai fixé ça après. :)